requirements/base.in

```diff
@@ -1,4 +1,4 @@
 # Core requirements for using this application
 
 Django>=1.8,<2.0          # Web application framework
-
+django-waffle==0.12.0
```

Comment:
Maybe add a comment that this is pinned to match edx-platform?

Reply:
You got it